starter-code/functions-and-arrays.js

```diff
@@ -20,21 +20,45 @@ var words = [
   "orchard",
   "crackpot"
 ];
+
+
+function findLongestWord (words) {
+  var j = "";
+    for(var i=0; i < words.length; i++){
+        if( words[i].length > j.length) {
+            j = words[i];
+          }
+    }
+
+  return j
+}
+
+
+
 var longest = findLongestWord(words);
 console.log(longest);
 
 // Calculating a Sum
 function sumArray (array) {
-
+  var j = 0;
+  for(var i = 0; i < numbers.length; i++) {
+    j = j + numbers[i];
+  }
+  return j
 }
 
 var numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 var total = sumArray(numbers);
 console.log(total);
 
+
 // Calculate the Average
 function averageNumbers (array) {
-
+  var j = 0;
+  for(var i = 0; i < numbers.length; i++) {
+    j = j + numbers[i];
+  }
+  return j/numbers.length
```

Comment:
Your solution is good, but since you created a function to calculate the sum of an array, you could call this function and divide the result by the array.length